src/shared/cpu-set-util.h

```diff
@@ -58,6 +58,7 @@ typedef struct NUMAPolicy {
         /* Always use numa_policy_get_type() to read the value */
         int type;
         CPUSet nodes;
+        bool derive_cpu_affinity_from_numa_mask;
```

Comment:
shouldn't that be part of ExecContext rather than NUMAPolicy?

Reply:
I feel like this should stay here but if you feel strongly about this (i.e. will block merging) I will move it to `ExecContext`.

Reply:
id'd prefer it in ExecContext I think